app/models/audit/AuditTaskTable.scala

```diff
@@ -301,6 +306,33 @@ object AuditTaskTable {
     tasksWithLabels.list.map(x => AuditTaskWithALabel.tupled(x))
   }
 
+  /**
+    * Return a list of missions for a specific user
+    *
+    * @param userId User id
+    * @return
+    */
+  def selectMissions(userId: UUID): List[AuditMission] = db.withSession { implicit session =>
```

Comment:
This function probably belongs in MissionTable.scala instead of AuditTaskTable.scala. Also could you add a couple small comments?

Reply:
Moved code to MissionTable and added comments